.github/workflows/crawler.yml

```diff
@@ -3,10 +3,21 @@
 name: Crawler
 
 on:
+  push:
+    branches: [ master, main ]
+    tags: [ v* ]
+  pull_request:
   schedule:
   - cron: '0 14 * * *'
   workflow_dispatch:
+    inputs:
+      args:
+        description: Extra args to pass to crawler.rb script
```

Comment:
The description should tell a user what the argument is used for and what values are accepted. For example: 'To reload all the document enter "force"'. Please change the description.

Reply:
Updated